src/lv_widgets/lv_win.h

```diff
@@ -57,6 +57,7 @@ typedef struct {
     lv_obj_t * header;                /*Pointer to the header container of the window*/
     char * title_txt;                 /*Pointer to the title label of the window*/
     lv_coord_t btn_w;                 /*Width of the control buttons*/
+    uint8_t title_txt_align;          //Control the alignment of the header text
```

Comment:
Please use `/* */` type comments

Reply:
Sure, but why do you prefer this type of comment: `/* */`

Reply:
We used to support C89 which required this type of comment. Due to consistency, it's stuck around as part of the project's [coding style](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md#comments).